manifests/squid.pp

```diff
@@ -47,7 +71,7 @@
     order          => '60',
     config_entries => {
       'cache_peer'          => "${streamer_host} parent ${streamer_port} 0 no-digest no-query originserver name=PulpStreamer",
-      'cache_peer_access'   => 'PulpStreamer allow all',
+      'cache_peer_access'   => 'PulpStreamer allow localhost',
```

Comment:
This looks like a general thing that can be applied to all installations, correct? Only Apache/Pulp connect to squid.

Reply:
yes. I think so.